api/apps/api/src/modules/access-control/role.api.entity.ts

```diff
@@ -6,12 +6,15 @@ import { Entity, PrimaryColumn } from 'typeorm';
  * roles are added to the database.
  */
 export enum Roles {
+  organization_admin = 'organization_admin',
   organization_owner = 'organization_owner',
+  organization_user = 'organization_user',
   project_owner = 'project_owner',
-  organization_admin = 'organization_admin',
   project_contributor = 'project_contributor',
-  organization_user = 'organization_user',
   project_viewer = 'project_viewer',
+  scenario_owner = 'scenario_owner',
+  scenario_contributor = 'scenario_contributor',
+  scenario_viewer = 'scenario_viewer',
```

Comment:
Migration may be required to update the enum on database.

Reply:
@kgajowy [47290cd](https://github.com/Vizzuality/marxan-cloud/pull/709/commits/47290cda6be490f94b08ba94647ed0aa095f00f6)

I also fixed a little punctuation error there in the following commit.